agent/bench-scripts/pbench-fio

```diff
@@ -551,6 +552,11 @@ function fio_run_job() {
 			ssh $ssh_opts $client "pushd $benchmark_results_dir >/dev/null; screen -dmS fio-server bash -c ''$bench_cmd' --server 2>&1 >client-result.txt'"
 		done
 		wait
+		debug_log "waiting for fio process(server) to start on clients"
+		for client in `echo $clients | sed -e s/,/" "/g`; do
+			timeout 60 bash -c 'until printf "" 2>>/dev/null >>/dev/tcp/$0/$1; do sleep 1; done' $client $fio_server_port
```

Comment:
Can we check the return status of `timeout` for the value of `124` so that we can emit an error message saying the server port was never opened?

Also, can you add the `--kill-after=1` parameter to ensure nothing is left around?

Reply:
Because in my tests, fio server takes always between 30 and 50 seconds. Also because I thought 60s was a reasonable amount of time to wait.